custom/icds_reports/ucr/data_sources/dashboard/remove_member_form.json

```diff
@@ -0,0 +1,103 @@
+{
+  "domains": [
+    "icds-dashboard-qa",
+    "sankalp_cas",
+    "zohaib-sandbox",
+    "akshita-sandbox",
+    "sunaina-sandbox",
+    "laurence-project-1",
+    "jessica-icds-cas",
+    "marissa-test",
+    "derek-icds-sandbox",
+    "priyanka-app",
+    "shrena-dev",
+    "aparatest",
+    "reach-sandbox",
+    "reach-dashboard-qa",
+    "reach-test",
+    "icds-cas",
+    "cas-lab"
+  ],
+  "server_environment": [
+    "india",
+    "icds"
+  ],
+  "config": {
+    "table_id": "static-remove_member",
+    "display_name": "Forms - Remove Member (REACH Dashboard)",
+    "referenced_doc_type": "XFormInstance",
+    "base_item_expression": {},
+    "configured_filter": {
+      "type": "boolean_expression",
+      "operator": "eq",
+      "expression": {
+        "type": "property_name",
+        "property_name": "xmlns"
+      },
+      "property_value": "http://openrosa.org/formdesigner/705d02eb71b1e3f493ecdb519f84b8c9cf9f5fbf"
+    },
+    "configured_indicators": [
+      {
+        "column_id": "reason_closure",
+        "datatype": "string",
+        "type": "expression",
+        "expression": {
+            "type": "property_path",
+            "property_path": ["form", "reason_closure"]
+        }
+      },
+      {
+        "column_id": "person_case_id",
+        "datatype": "string",
+        "type": "expression",
+        "expression": {
+          "type": "property_path",
+          "property_path": ["form", "case_load_person_0", "case", "@case_id"]
+        }
+      },
+      {
+        "display_name": "Supervisor ID",
+        "transform": {},
+        "datatype": "string",
+        "expression": {
+          "type": "ancestor_location",
+          "location_id": {
+            "type": "named",
+            "name": "user_location_id"
+          },
+          "location_type": "supervisor",
+          "location_property": "_id"
+        },
+        "is_primary_key": true,
+        "is_nullable": false,
+        "column_id": "supervisor_id",
+        "type": "expression"
+      }
+    ],
+    "named_expressions": {
+      "user_location_id": {
+        "type": "icds_user_location",
+        "user_id_expression": {
+          "type": "root_doc",
+          "expression": {
+            "type": "property_path",
+            "property_path": [
+              "form",
+              "meta",
+              "userID"
+            ]
+          }
+        }
+      }
+    },
+    "named_filters": {},
+    "engine_id": "icds-ucr-citus",
+    "sql_settings": {
+      "citus_config": {
+        "distribution_type": "hash",
+        "distribution_column": "supervisor_id"
+      },
+      "primary_key": ["supervisor_id","doc_id"]
```

Comment:
It is good to have index on the time end column so that filter can be faster. I think you can see the example in some of our Dashboard UCR in `data_sources>dashboard` folder.

Reply:
done